src/init.cpp

```diff
@@ -29,6 +29,7 @@
 
 namespace dlaf {
 std::ostream& operator<<(std::ostream& os, const configuration& cfg) {
+  os << "  print_config = " << cfg.print_config << std::endl;  // TODO: Is this silly?
```

Comment:
This will always print 1 since it'll only be printed when it's 1... Leave it out?

Reply:
Yeah, I'd ditch it.